### DISCUSSION TITLE
speech regulator TTS

### DISCUSSION THREAD

**Discussion by jeffersonquispe** [NVIDIA-NeMo/NeMo#4403] [Q&A] [answered]
Has anyone tried how to modify the speaker's speed without losing pitch? Is there any additional configuration for the Fastpitch and HifiGan models?

**Comment by ericharper** [comment]
@redoctopus could you take a stab at this one?

**Comment by redoctopus** [comment, marked as answer]
We do have "pace" for FastPitch, which you can pass in as an argument in `generate_spectrogram()`. It defaults to 1.0, but you can tweak it up or down as needed.
